app-arch/createrepo_c/metadata.xml

```diff
@@ -9,6 +9,10 @@
 		<email>sam@gentoo.org</email>
 		<name>Sam James</name>
 	</maintainer>
+	<use>
+		<flag name="drpm">Enable delta RPM support via <pkg>app-arch/drpm</pkg></flag>
+		<flag name="legacy">Build with support for legacy weakdeps and hashes (not recommended!)</flag>
```

Comment:
I assume you actually needed this in the wild, hence had to add an option for it? (Otherwise I'd say we just shouldn't do it at all)

Reply:
tbh `drpm` could probably be unconditionally enabled - I can't see that enabling having delta RPM support will break anything.

The `legacy` flag enables both legacy suse/mandriva weakdeps support (as opposed to the 'modern' weakdeps support, I'm assuming - basically the equivalent of ebuild optfeature) and the use of SHA1 & MD5 for digests. I can see a use case where using the modern createrepo_c implementation to manage legacy RPM repositories is desirable - there are some edge cases in the legacy script for basic functionality. I think that I'd prefer that it remain toggleable for now, eventually legacy distro releases will age out.

If you have a strong preference for getting rid of the 'legacy' flag I'm happy remove it; I don't think that *my* use case is quite old enough to need it, it's just within throwing distance!